server/api/room.ts

```diff
@@ -304,17 +304,9 @@ const undoEvent: RequestHandler<{ name: string }> = async (req, res) => {
 };
 
 const addVote: RequestHandler<{ name: string }, unknown, OttApiRequestVote> = async (req, res) => {
-	if (!req.token) {
```

Comment:
`req.token` is not on the body. You shouldn't be touching auth tokens with these validation PRs